ui/installer/VCSA/configs

```diff
@@ -1,16 +1,30 @@
-# Enter the IP of the server running VCSA
+# Enter the IP of VCSA
 VCENTER_IP=""
 
-# Update the following lines if you have a local web server that is accessible from your VCSA and VIC UI zip files are uploaded to it
+# Change the VIC_UI_HOST_URL variable to the URL to the path where
+# the plugin zip files are located. For example, if com.vmware.vic-v1.1.0.zip is
+# available at https://192.168.1.5/com.vmware.vic-v1.1.0.zip,
+# simply type in https://192.168.1.5/
+#
+# NOTE: Installing the plugin by setting VIC_UI_HOST_URL to "NOURL" is deprecated
+# and therefore offically NOT supported. Please use it at your own risk.
+VIC_UI_HOST_URL=""
 
-# Change the following line to the full URL of the path where the zip files are located
-# For example, if vic-ui.zip is available at https://192.168.1.5/vic-ui-plugins/vic-ui.zip, type in: https://192.168.1.5/vic-ui-plugins/
-# Note: each uploaded zip file will be accessed only once when the user logs into vSphere Web Client for the first time after successful installation
-VIC_UI_HOST_URL="NOURL"
-
-# If VIC_UI_HOST_URL starts with "https", enter SHA-1 thumbprint of the web server hosting the vic-ui.zip file
-# The fingerprint should be in the format of A0:B1:C2:... In case you retrieved the fingerprint from Google Chrome make sure to replace spaces with colons (:)
```

Comment:
You may want to leave this line in just in case : "The fingerprint shoudl be in the format of A0:B1:C2...."  It proved helpful to me when I was running the setup.